BuildSystem/src/Build.py

```diff
@@ -481,6 +481,23 @@ def makeInjectionsInfo(compiledCodes: Library):
     else:
         return data
 
+def buildSDCard():
+    outputfiles = [f for f in listdir('.\\Output') if isfile(join('.\\Output', f))]
```

Comment:
`\\` can be replaced with `/` to better support other platforms. (will still work on windows)